src/test/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorTest.java

```diff
@@ -62,16 +62,6 @@ public void shouldStartAndStop() {
     connector.stop();
   }
 
-
-  @Test
```

Comment:
Why are we dropping this test? Can we do an incremental change instead?

Reply:
again, not sure how to mock the client here and this does not test much

Reply:
We should be able to add a test since you can pass a `clientFactory` now.